src/FirebaseObject.js

```diff
@@ -151,7 +151,7 @@
         /**
          * Listeners passed into this method are notified whenever a new change is received
          * from the server. Each invocation is sent an object containing
-         * <code>{ type: 'updated', key: 'my_firebase_id' }</code>
+         * <code>{ type: 'child_updated', key: 'my_firebase_id' }</code>
```

Comment:
this should be `value` https://github.com/firebase/angularfire/blob/jw-watch-docs/src/FirebaseObject.js#L244